docs/DataFormats.md

```diff
@@ -10,13 +10,13 @@ DIVE Desktop and Web support a number of annotation and configuration formats.
 * DIVE Annotation JSON (default annotation format)
 * DIVE Configuration JSON
 * VIAME CSV
-* COCO and KWCOCO
-* KPF (KWIVER Packet Format) for MEVA
+* COCO and KWCOCO (web only)
+* KPF (KWIVER Packet Format) for MEVA (web only)
```

Comment:
should we advertise this when we don't have a user facing UI to load this.  Or should we explain how this is done?
Unless I missed something, I saw some export of .yml/KPF extensions but know actual usage.

Reply:
There is UI for it now.   It just requires you to put all your KPF into a single yml file.  There's actually nothing about KPF that requires 3 files, and you can concatenate all like this.

```
cat  2018-03-07.11-05-07.11-10-07.school.G339.*.yml > combined.yml
```

I'm not really willing to modify all this UI to accept multiple annotation files to support a format that's already deprecated, so this is my compromise.